content/how-to/account-management/change-account-information/index.md

```diff
@@ -72,29 +83,21 @@ provide the following information:
 
 ### Best practices for legal change forms
 
-The Service Transfer and Primary Contact Change forms have many required
-fields that must be complete before the Billing team can process the forms.
+The **Service Transfer change forms** and **Primary Contact Change forms** have many required
+fields that must be completed before the Billing team can process the forms.
 After you have completed the form, scan the document and attach it to the
 support ticket in the [Cloud Control Panel](https://login.rackspace.com) or
 the [MyRackspace Control Panel](https://login.rackspace.com).
 
-To avoid form rejection, use the following tips when completing the
+To avoid form rejection, use the following tips while completing the
 form:
 
 - The signatures on the **Service Transfer** form *must* come
   from an officer or an owner of the company.
-- The owner or officer must initial any changes regarding an increase
+- The owner or officer must initiate any changes regarding an increase
```

Comment:
they really may have meant "initial" as in signature or to sign off. We may want to put this back to "initial".